### ISSUE TITLE
How to build the Python module tflite-support-0.4.4 using distutils?

### ISSUE THREAD

**Issue by yurivict** [tensorflow/tflite-support#991]
For the 0.1.0a1 version [pypi.org had the source tarball](https://pypi.org/project/tflite-support/).
Now there is no source tarball for the version 0.4.4.
When the setup.py file from this repository is use it only builds this file: ```bin/tflite_codegen```.

How to build the Python module tflite-support-0.4.4 correctly?
Could you please consider adding the source tarball back to pypi.org?

**Comment by littlebee** [comment]
Well should I read the obit for tflite-support or should you?   Maybe google is waiting until AI can fix the issues with tflite-support.

**Comment by glenn-jocher** [comment]
Agreed. We don't know how to work with this package in 3.12 in Ultralytics. We're seeking to remove it from our dependencies now.
